integration-tests/test-extension/src/main/resources/application.properties

```diff
@@ -6,3 +6,98 @@ quarkus.log.file.format=%d{HH:mm:ss} %-5p [%c{2.}]] (%t) %s%e%n
 
 # Resource path to DSAPublicKey base64 encoded bytes
 quarkus.root.dsa-key-location=/DSAPublicKey.encoded
+
+### Configuration settings for the TestBuildTimeConfig config root
```

Comment:
If it's a test fix, should it be squashed in the previous commit touching the test extension?

Or it's related to some later commits?

Reply:
Well - if the requirement is that every commit is perfectly non breaking, then pretty much this whole PR has to be squashed to one commit... I thought clarity was probably more important.